readthedocs/telemetry/collectors.py

```diff
@@ -89,6 +90,37 @@ def collect(self):
                 "all": all_apt_packages,
             },
         }
+        data["doctool"] = self._get_doctool()
+        return data
+
+    def _get_doctool_name(self):
+        if self.version.is_sphinx_type:
+            return "sphinx"
+
+        if self.version.is_mkdocs_type:
+            return "mkdocs"
+
+        return "generic"
+
+    def _get_doctool(self):
+        data = {
+            "name": self._get_doctool_name(),
+            "extensions": [],
+            "html_theme": "",
+        }
+
+        if self._get_doctool_name() != "sphinx":
+            return data
+
+        conf_py_dir = os.path.join(
+            self.checkout_path,
+            os.path.dirname(self.config.sphinx.configuration),
+        )
+        filepath = os.path.join(conf_py_dir, "_build", "json", "telemetry.json")
+        if os.path.exists(filepath):
+            with open(filepath, "r") as json_file:
+                content = json_file.read()
+            data.update(self._safe_json_loads(content, {}))
```

Comment:
Should we filter out the ones that are the default sphinx extensions? I guess it's not a ton of data, but could be useful to filter out here instead of downstream.

Reply:
I updated the extension to only save `app.config.extensions` which are only the ones defined by the user. See [readthedocs/readthedocs-sphinx-ext@`f9cc1bd` (#117)](https://github.com/readthedocs/readthedocs-sphinx-ext/pull/117/commits/f9cc1bdb237257f52d2a6c279c5c15d5f967fe6f)